components/autofill/src/db/credit_cards.rs

```diff
@@ -231,7 +233,7 @@ mod tests {
         Ok(guids)
     }
 
-    fn clear_tables(conn: &Connection) -> rusqlite::Result<(), rusqlite::Error> {
+    pub fn clear_tables(conn: &Connection) -> rusqlite::Result<(), rusqlite::Error> {
```

Comment:
This function may actually be called in other places, but I only see it being used in the engine code. If that's the case, how do you feel about moving the function there? Oh, and also renaming to `clear_credit_card_tables` for clarity's sake.